package.json

```diff
@@ -36,13 +53,17 @@
     "react-dom": "^16.12.0",
     "react-draggable": "^4.4.3",
     "react-tippy": "^1.3.4",
+    "regenerator-runtime": "^0.13.5",
     "three": "^0.108.0",
     "uuid": "^8.0.0",
     "ws": "^7.3.0"
   },
   "devDependencies": {
-    "@rollup/plugin-json": "^4.1.0",
```

Comment:
Please add back this removed dependency.

Reply:
All set w/ latest commit.